provisioning/miniprov/hare_mp/main.py

```diff
@@ -53,8 +58,27 @@ def execute(cmd: List[str]) -> str:
     return out
 
 
-def setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+def create_logger_directory():
+    """Create log directory if not exsists."""
+    _logger_directory = os.path.join(LOG_DIR)
+    if not os.path.isdir(_logger_directory):
+        try:
+            os.mkdir(_logger_directory)
```

Comment:
This will work if parent directory (`/var/log/seagate/`) already exists. Can it happen that it doesn't exist? If yes, then probably [`os.makedirs`](https://docs.python.org/3/library/os.html#os.makedirs) should be considered instead

Reply:
Now I am trying without parent directory, but this function not creating any directory for it. So, If we want create any directory into parent directory then os.makedirs should be considered.

Reply:
Yeah I think `makedirs` is a better option anyway, it is similar to `mkdir -p`.

Reply:
done